src/plugins/cosmos/components/CosmosActionButtons/CosmosActionButtons.tsx

```diff
@@ -1,53 +1,68 @@
+import { Button } from '@chakra-ui/react'
 import { Asset } from '@shapeshiftoss/types'
-import { StakingAction } from 'plugins/cosmos/components/modals/Staking/Staking'
+import { StakeRoutes } from 'plugins/cosmos/components/modals/Staking/Staking'
+import { useHistory } from 'react-router-dom'
 import { Text } from 'components/Text'
-import { useModal } from 'context/ModalProvider/ModalProvider'
 
 type CosmosActionButtonsProps = {
-  activeAction: StakingAction
+  activeRoute: StakeRoutes
   asset: Asset
 }
 export const CosmosActionButtons = ({
-  activeAction,
+  activeRoute,
   asset,
   ...styleProps
 }: CosmosActionButtonsProps & FlexProps) => {
-  const { cosmosStaking } = useModal()
+  const history = useHistory()
 
-  const handleUnstakeClick = () => {
-    cosmosStaking.open({ assetId: asset.caip19, action: StakingAction.Unstake })
+  const handleClick = (route: StakeRoutes) => {
+    history.push(route)
   }
 
-  const handleStakeClick = () => {
-    cosmosStaking.open({ assetId: 'cosmoshub-4/slip44:118', action: StakingAction.Stake })
+  const isStakeAction = () => {
+    return (
+      activeRoute !== StakeRoutes.Stake &&
+      activeRoute !== StakeRoutes.StakeConfirm &&
+      activeRoute !== StakeRoutes.StakeBroadcast
+    )
+  }
+
+  const isUnstakeAction = () => {
+    return (
+      activeRoute !== StakeRoutes.Unstake &&
+      activeRoute !== StakeRoutes.UnstakeConfirm &&
+      activeRoute !== StakeRoutes.UnstakeBroadcast
+    )
   }
 
-  const bgColor = useColorModeValue('gray.50', 'gray.850')
   return (
-    <Flex width='100%' bgColor={bgColor} borderRadius='12px' {...styleProps}>
+    <Flex width='100%' borderRadius='12px' my='12px' justifyContent='center' {...styleProps}>
+      <Button
+        colorScheme='blue'
+        variant={activeRoute !== StakeRoutes.Overview ? 'ghost' : 'ghost-filled'}
+        onClick={() => handleClick(StakeRoutes.Overview)}
+        mx='2px'
+        isDisabled={false}
+      >
+        <Text translation='defi.overview' fontWeight='normal' />
+      </Button>
       <Button
-        flexGrow={1}
         colorScheme='blue'
-        isActive={activeAction === StakingAction.Stake}
-        variant={activeAction === StakingAction.Unstake ? 'ghost' : undefined}
-        onClick={handleStakeClick}
+        variant={isStakeAction() ? 'ghost' : 'ghost-filled'}
+        onClick={() => handleClick(StakeRoutes.Stake)}
         isDisabled={false}
+        mx='2px'
       >
-        <Text translation={['defi.stakeAsset', { assetSymbol: asset.symbol }]} fontWeight='bold' />
+        <Text translation='defi.stake' fontWeight='normal' />
       </Button>
       <Button
-        isActive={activeAction === StakingAction.Unstake}
         colorScheme='blue'
-        onClick={handleUnstakeClick}
-        flexGrow={1}
-        variant={activeAction === StakingAction.Stake ? 'ghost' : undefined}
+        onClick={() => handleClick(StakeRoutes.Unstake)}
```

Comment:
Note: after your refactor `handleClick` into separate, explicit methods, you won't need an anonymous function in the `onClick`s props anymore, don't forget to remove these